stages/apps/sharedsvc/harbor.tf

```diff
@@ -0,0 +1,35 @@
+locals {
+  ingress_annotations = {
+    "nginx.ingress.kubernetes.io/force-ssl-redirect" : true
+    "nginx.ingress.kubernetes.io/proxy-body-size" : "0"
+    "kubernetes.io/ingress.class" : module.nginx_external.ingress_class
+  }
```

Comment:
same as above, I think these should be moved to a more generic place or renamed so it's understood that these are specifically for harbor